documentation/src/main/asciidoc/topics/con_xsite_backup_strategies.adoc

```diff
@@ -0,0 +1,34 @@
+[id='xsite_backups-{context}']
+= Backup Strategies
+{brandname} clusters can use different strategies for backing up data to remote sites.
+
+{brandname} replicates across sites at the same time that writes to local caches occur. For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+
+.Synchronous backups
+When {brandname} applies changes to the remote site, it waits until the operation completes.
+
+The synchronous strategy lets you define policies that control how {brandname} handles writes if backup operations fail.
+
+.{brandname} failure policies
+|===
+| Policy | If backup operations fail...
+
+| `IGNORE`
+| {brandname} ignores exceptions and writes to the local cache.
+
+| `WARN`
+| {brandname} logs exceptions and writes to the local cache. This is the default failure policy.
+
+| `FAIL`
+| {brandname} throws exceptions and aborts writes to the local cache.
+|===
```

Comment:
isn't the CUSTOM failure policy missing?

Reply:
@pruivo I didn't add CUSTOM here on purpose. it's in the schema so users can figure it out there. in the docs I mention only the Infinispan failure policies. I guess we could mention the fact that you can implement custom failure policies but it seems a bit like overkill imho. we shouldn't try to get absolutely everything in the docs.